packages/@aws-cdk/custom-resources/lib/provider-framework/provider.ts

```diff
@@ -172,7 +172,7 @@ export class Provider extends CoreConstruct implements cfn.ICustomResourceProvid
     const fn = new lambda.Function(this, `framework-${entrypoint}`, {
       code: lambda.Code.fromAsset(RUNTIME_HANDLER_PATH),
       description: `AWS CDK resource provider framework - ${entrypoint} (${this.node.path})`.slice(0, 256),
-      runtime: lambda.Runtime.NODEJS_10_X,
+      runtime: lambda.Runtime.NODEJS_12_X,
```

Comment:
and here

Reply:
Reverted and created issue for follow-up https://github.com/aws/aws-cdk/issues/13225